src/panels/lovelace/sections/hui-section.ts

```diff
@@ -26,6 +26,13 @@ import { parseLovelaceCardPath } from "../editor/lovelace-path";
 import { generateLovelaceSectionStrategy } from "../strategies/get-strategy";
 import type { Lovelace } from "../types";
 import { DEFAULT_SECTION_LAYOUT } from "./const";
+import { fireEvent } from "../../../common/dom/fire_event";
+
+declare global {
+  interface HASSDomEvents {
+    "section-visibility-changed": { value: boolean };
+  }
+}
```

Comment:
declare these at the bottom of the file (there is already a declaration there for global)

Reply:
We used to put events at the top and component name at the bottom but that's a good feedback. May be we should move events to the bottom for every component. I prefer to keep it at the top for consistency and I will look if it's a good practice to group them.